src/vs/workbench/api/common/extHostCommands.ts

```diff
@@ -43,6 +47,11 @@ export class ExtHostCommands implements ExtHostCommandsShape {
 		logService: ILogService
 	) {
 		this._proxy = mainContext.getProxy(MainContext.MainThreadCommands);
+		this._onDidExecuteCommand = new Emitter<ICommandEvent>({
+			onFirstListenerDidAdd: this._proxy.$onDidExecuteCommand,
+			onLastListenerRemove: this._proxy.$disposeListener,
```

Comment:
Not sure but those two might miss a bind-call or change to `onFirstListenerDidAdd: () => this._proxy.$onDidExecuteCommand()` (and corresponding)